site/content/contribute/getting-started/core-committers.md

```diff
@@ -158,6 +158,7 @@ Below is the list of core committers working on Mattermost:
 - **<a name="caleb.roseland">Caleb Roseland</a>**
     - @caleb.roseland on [community.mattermost.com](https://community.mattermost.com/core/messages/@caleb.roseland) and [@calebroseland](https://github.com/calebroseland) on GitHub
     - Dev areas: Desktop / Web apps
+- **<a name="avasconcelos114">Andre Vasconcelos</a>**
     - @avasconcelos114 on [community.mattermost.com](https://community.mattermost.com/core/messages/@avasconcelos114) and [@avasconcelos114](https://github.com/avasconcelos114) on GitHub
     - Dev areas: Web App, Mobile Apps
```

Comment:
Mobile App**s**?

Reply:
Yeah, I've seen it referenced as apps (since it's mobile apps for iOS and Android).